docs/releases/3.0.0.rst

```diff
@@ -0,0 +1,68 @@
+Translate Toolkit 3.0.0
+***********************
+
+*Released on 15 June 2020*
+
+This release contains improvements and bug fixes.
+
+
+Changes
+=======
+
+Formats and Converters
+----------------------
+
+- l20n
```

Comment:
This section is usually sorted with the most widely used formats first. This should probably be the last one.